Orange/data/tests/test_variable.py

```diff
@@ -657,8 +657,8 @@ class TestTimeVariable(VariableTest):
         ('2015-285', 1444608000, '2015-10-12'),
         ('2015-10', 1443657600, '2015-10-01'),
         ('2015', 1420070400, '2015-01-01'),
-        ('01:01:01.01', 3661.01, '01:01:01.010000'),
-        ('010101.01', 3661.01, '01:01:01.010000'),
+      #  ('01:01:01.01', 3661.01, '01:01:01.010000'),
+      #  ('010101.01', 3661.01, '01:01:01.010000'),
```

Comment:
Since now, we have avoided pandas 2.0.0; I think it is not necessary to comment out those
```suggestion
        ('01:01:01.01', 3661.01, '01:01:01.010000'),
        ('010101.01', 3661.01, '01:01:01.010000'),
```

Reply:
True. I will remove that.